packages/react/src/docs/customization.stories.mdx

```diff
@@ -0,0 +1,108 @@
+import { Canvas, Meta, Story, Source, ArgsTable } from '@storybook/addon-docs';
+
+<Meta title="Getting Started/Customization"/>
+
+# Customization
+
+At some point you might want to customize the design system's colors or the look and feel of some components. This is totally possible, and we will explain how in this documentation.
+But before we start, we will need to explain the concept of **design tokens**.
+
+
+## What are design tokens ?
+
+Design tokens are the fundamental variables defining the precise behavior and rendering of ui components.
+
+For example it could be:
+
+- The primary color of a text element
+- The standard spacing between two elements
+- The border radius of a button
+- ...
+
+Design tokens are the building blocks of a design system. They are the smallest pieces of a design system, and they are used to build components.
+
+## How to customize design tokens ?
+
+Cunningham makes possible to create a configuration file at the root of your project, which will be used to customize the design tokens.
+
+### Typescript
+
+If you prefer using Typescript, create a file named `cunningham.ts` at the root of your project with the following content:
+
+<Source dark={true} code={`
+// cunningham.ts
+export \default {
+  theme: {
+    colors: {
+      "primary-500": "purple",
+    },
+  },
+};`}></Source>
+
+### Javascript
+
+Or if you prefer Javascript, create a file named `cunningham.cjs` at the root of your project with the following content:
+
+<Source dark={true} code={`
+// cunningham.cjs
+module.exports = {
+  theme: {
+    colors: {
+      "primary-500": "purple",
+    },
+  },
+};`}></Source>
+
+In this configuration file you can overwrite all the default values of the design system.
+**You can find the default values [here](https://github.com/openfun/cunningham/blob/main/packages/tokens/src/bin/cunningham.ts).**
+
+### Build
+
+If you followed the previous steps, you can now build the design tokens by running the following command:
```

Comment:
```suggestion
If you followed the previous steps, you can now rebuild the design tokens by running the following command:

```

I'll use `rebuild` instead of `build` as we assume developer previously build tokens during installation step ?